Carbamazepine-Model.json

```diff
@@ -10681,8 +10708,7 @@
               "Value": 0.243,
               "ValueOrigin": {
                 "Source": "Publication",
-                "Method": "InVitro",
-                "Description": "Morselli 1975"
+                "Description": "Morselli 1975 (DOI: 10.1007/978-3-642-85921-2_16)"

```

Comment:
Method was inVitro and now empty - on purpose?

Reply:
Difficult to classify in my opinion; blood was sampled *in vivo* and further processed and analyzed *in vitro*, not sure how to best call that...